package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -866,8 +865,8 @@ def __init__(self, universe, selection1='protein',
             :attr:`~DEFAULT_ACCEPTORS` values.
             ["CHARMM27"]
         donors : sequence (optional)
-            Extra H donor atom types (in addition to those in
-            :attr:`~DEFAULT_DONORS`), must be a sequence.
+            Extra H donor atom types (The heavy atom not the hydrogen)
+            (in addition to those in :attr:`~DEFAULT_DONORS`), must be a sequence.
```

Comment:
I'll be honest I have trouble understanding what this means, in part because there's two sets of text in brackets one after the other. Is there any way to make this a bit clearer?

Possibly linked to this, would it be worth adding something in the docstring examples or notes about it?